shell/basic.sh

```diff
@@ -49,7 +49,29 @@ case $fruit in
 esac
 
 # TODO: Add json parsing
+
+
 # TODO: Add grep
+    # grep_example.txt is an example file you can create your own file and give its name instead of grep_example.txt to search in that file.
+grep "grep" grep_example.txt  # change the keyword inside the bracket to search for differnt words it will print the whole line if the given keyword is in that lin.
```

Comment:
```suggestion
grep "grep" grep_example.txt  # change the keyword inside the bracket to search for different words it will print the whole line if the given keyword is in that line.
```